presto-main/src/main/java/com/facebook/presto/sql/planner/StatisticsAggregationPlanner.java

```diff
@@ -52,7 +53,7 @@
 
 public class StatisticsAggregationPlanner
 {
-    private final PlanVariableAllocator variableAllocator;
+    private final VariableAllocator variableAllocator;
```

Comment:
unrelated change?

Reply:
Good catch. Its not completely unrelated. I was planning to move StatisticsAggregationPlanner and LogicalPlanner to leverage only VariableAllocator instead of PlanVariableAllocator. Which is something we will have to eventually.
I did not make those changes completely. I will revert this part and make the changes to leverage VariableAllocator instead of PlanVariableAllocator separately.